uitools/toolkitwidgets/CMakeLists.txt

```diff
@@ -11,77 +11,76 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
See other comment. I think we should go higher and match the minimum listed by Qt for 6.8 https://doc.qt.io/qt-6/cmake-supported-cmake-versions.html.